flycheck.el

```diff
@@ -8630,6 +8631,16 @@ See URL `http://slim-lang.com'."
           line-end))
   :modes slim-mode)
 
+(flycheck-define-checker slim-lint
+  "A SLIM linter
+
+See URL `https://github.com/sds/slim-lint'."
+  :command ("slim-lint" "--reporter=checkstyle"
+            source-original)
```

Comment:
Must this be `source-original`? Wouldn't `source` (or even `source-inplace`) work?